vendor-extra/LiberoPageBundle/src/EventListener/BuildView/ItemRefTeaserListener.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoPageBundle\EventListener\BuildView;
+
+use FluentDOM;
+use FluentDOM\DOM\Element;
+use GuzzleHttp\ClientInterface;
+use Libero\ViewsBundle\Views\LazyView;
+use Libero\ViewsBundle\Views\SimplifiedViewConverterListener;
+use Libero\ViewsBundle\Views\TemplateView;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Psr\Http\Message\ResponseInterface;
+
+final class ItemRefTeaserListener
+{
+    use SimplifiedViewConverterListener;
+
+    private $client;
+    private $converter;
+
+    public function __construct(ClientInterface $client, ViewConverter $converter)
```

Comment:
mixing the `Listeners` with `Clients` is the dangerous part in terms of layering, but I'd agree there's no other clear option here at the moment